src/styles/variables.scss

```diff
@@ -0,0 +1,3 @@
+  $max-month-length: 31;
+  $cell-size: 100px;
```

Comment:
Consider using a more descriptive variable name for the cell size, such as $day-cell-size.